### ISSUE TITLE
add json support

### ISSUE THREAD

**Issue by agauniyal** [agauniyal/isaac-core#10]
Will add once I get sorted benchmarks, probably going with https://github.com/nlohmann/json though, since API is quite nice. Might replace with `rapidjson` later for perf reasons.

- [x] Add json lib to meson wrap db
- [x] Setup build commands to import json lib

**Comment by agauniyal** [comment]
On hold till @hargurpalsingh & @anjanikumarsharma provide sheet on benchmarks given to them.

**Comment by agauniyal** [comment]
I've opened following PR - https://github.com/mesonbuild/json/pull/1

**Comment by agauniyal** [comment]
Blocked on - https://github.com/mesonbuild/meson/issues/759
